core/utils/poly-look/src/css/themes.css

```diff
@@ -0,0 +1,21 @@
+:root,
+.poly-theme {
```

Comment:
Since this is apparently going to be used in combination with other themes, but it's not a theme *per se*, maybe call it poly-root? or poly-base? Finding it in combinations such as `poly-theme poly-theme-dark` is kind of confusing (and maybe a bit iterative)

Reply:
Other companies will have other font-families and other line heights etc. This is the poly-theme, it's what we use everywhere. I think no other name will do. Earlier I had it like `poly-theme dark` but that was even more confusing so I went back to this. 

Imagine we would have styled sections for a company. You would just have to change that theme and everything changes

Reply:
Please take this only as a (possibly mine only) nit. When I see something like poly-theme and poly-theme-dark, since poly-theme is included in poly-theme-dark, I also think that styles are derived, and that somehow poly-theme-dark expands or overrides part of poly-theme. My suggestions were going in the direction of poly-theme being something that's going to be *composed* with poly-theme-dark, which is why I suggested poly-base and things like that. Could you consider something in that direction? Like renaming it "poly-theme-base" or "poly-theme-root" or something like that? That suggest it's in the same semantic field as poly-theme-dark, but does not have any kind of inclusion or hierarchical relationship with it.
I know, long rant, anyway, just a nit, as I say.